features/steps/steps.py

```diff
@@ -165,17 +165,16 @@ def step_impl(context, attribute, value):
     )
 
 
-@given('A file with {field} "{value}"')
-def step_impl(context, field, value):
+@given('A file with {field} "{values}"')
+def step_impl(context, field, values):
+    values = list(values.replace('"', "").split(', '))
```

Comment:
In @evandroAlfieri's guide we have "or". So instead of `split` on "," why don't we split on " or "? Instead of just blindly replacing `"` with ``. I'd say.

values = map(str.lower, map(lambda s: s.strip('"'), values.split(' or ')))

Reply:
That is better, yes. The rationale was that a good sentence would be "Given a file with Schema Identifier "IFC4X3", "IFC4X3_TC1" or "IFC4X3_ADD1". But maybe it is better to split on 'or', indeed. The sentence is would then be not completely perfect English ('a' or 'b' or 'c') but neither would the code I wrote  ('a','b','c'). The first one makes logical sense and is the closest to Evandro's guide. 

I only had to add the map object to a list to work with the rest of the function:
```suggestion
    values = list(map(str.lower, map(lambda s: s.strip('"'), values.split(' or '))))
```

Reply:
If we want to support "a, b or c". Then we have to use regexes.

```
>>> import re
>>> re.split(r' *(\bor\b|,) *', 'a, b, or c, d, or e')
['a', ',', 'b', ',', '', 'or', 'c', ',', 'd', ',', '', 'or', 'e']
>>> ps = _
>>> ps[1::2]
[',', ',', 'or', ',', ',', 'or']
>>> re.match(r'\,*or', "".join(ps[1::2]))
>>> ps = re.split(r' *(\bor\b|,) *', 'a, b, c, d, or e')
>>> re.match(r'\,*or', "".join(ps[1::2]))
<re.Match object; span=(0, 6), match=',,,,or'>
```

* zero or more spaces
* followed by
  - either `or` (at word boundaries)
  - or `,`
* followed by zero or more spaces

The second regex is to test we use zero or more `,` followed by one `or` to join the parts.

I'm not against it, it's fun, but not sure if this is worth the effort.

Reply:
That's also what I saw when I looked it up. I've used regexes before, but it tends to cost a lot of time (for me at least) to get it exactly right. 

Would it be okay for you to merge as in https://github.com/buildingSMART/ifc-gherkin-rules/pull/15/commits/b9c8689101207aa3f68d66c8b46e11274e9f0b66 ? 
I have time tomorrow, in the weekend here and there, and think it would be better spent on things more directly related to gherkin (fourth scenario needs to be adjusted, some documentation, test files, etc.) before Monday. I've discussed it briefly with Evandro and he shares this point of view.

Reply:
Ok from me :)